### DISCUSSION TITLE
Need Larger Files Upload directly with URL ? [Github Codespaces]

### DISCUSSION THREAD

**Discussion by MeetBhingradiya** [community/community#64309] [Codespaces] [answered]
### Select Topic Area

Question

### Body

I Don't Find any URL to upload. my file size is around 2.6+ GB, i have link.

**Comment by byrneh** [comment, marked as answer]
If you add a file to a repository via a browser, the file can be no larger than 25 MB.
If you attempt to add or update a file that is larger than 50 MiB, you will receive a warning from Git.
GitHub blocks files larger than 100 MiB.
To track files beyond this limit, you must use Git Large File Storage (Git LFS)
The GitHub Docs has some information on thia subject.
[working-with-files/managing-large-files/about-large-files-on-github](https://docs.github.com/en/repositories/working-with-files/managing-large-files/about-large-files-on-github)

**Reply by MeetBhingradiya** [reply]
That means i can't upload morethan 2 GB File
